.github/scripts/addUnblockedComment.js

```diff
@@ -0,0 +1,56 @@
+// When a blocking issue is closed:
+// 1. Leaves a comment on all the issues listed as blocked in body,
+
+module.exports = async ({ github, context }) => {
```

Comment:
```suggestion
/** @param {import('github-script').AsyncFunctionArguments} AsyncFunctionArguments */
module.exports = async ({ github, context }) => {
  const { repo, owner } = context.repo;

  const payload = /** @type {import('@octokit/webhooks-types').IssuesEvent} */ (context.payload);
module.exports = async ({ github, context }) => {
```

Can you please add types via JSDOC comments? See the other scripts in that directory for full examples